databuilder/extractor/dashboard/mode_analytics/mode_dashboard_extractor.py

```diff
@@ -13,10 +13,6 @@
 from databuilder.transformer.template_variable_substitution_transformer import \
     TemplateVariableSubstitutionTransformer, TEMPLATE, FIELD_NAME as VAR_FIELD_NAME
 
-# CONFIG KEYS
```

Comment:
Forgot to remove this. It's available at https://github.com/lyft/amundsendatabuilder/blob/master/databuilder/extractor/dashboard/mode_analytics/mode_dashboard_constants.py